helsingborg-stad/aterbruket-webapp
# Feature/style add page

Fixed some styling on add-page (more styling will come)
Added new categories

**To test:**
make sure to amplify push
start server and add a new item, to see the new styling and the new categories
